src/site/_includes/course.njk

```diff
@@ -55,8 +55,12 @@ inlineScripts:
             <div class="course-index font-mono">{{ loop.index0 | padStart(3, '0') }}</div>
           {% endif %}
         {% endfor %}
-        <h1 class="course-headline">{{ title }}</h1>
-        <p class="course-description">{{ description }}</p>
+        <h1 class="course-headline">{{ title | md | safe }}</h1>
```

Comment:
Is this disingenious because it doesn't work for the actual `<title></title>`, therefore we shouldn't do this here?

Reply:
I may be misunderstanding but I think we get the results that we want.

Using this PR, if I have the following frontmatter:

```yaml
title: Learn `<ol>` CSS
authors:
  - andybell
description: An evergreen `<ul>` CSS course and reference to level up your web styling expertise.
date: 2021-05-04
tags:
  - css
  - layout
```

I end up with:

```html
<title>Learn <ol> CSS</title>
<meta name="description" content="An evergreen <ul> CSS course and reference to level up your web styling expertise." />
```

And the page renders as expected:

![image](https://user-images.githubusercontent.com/1066253/131758570-fb0168c8-6e13-4fbf-9f81-7866eb6f0f0c.png)

Reply:
Although it's maybe not great to have an actual piece of html inside of the title because you could in theory wrote `</title>` and the page would blow up

Reply:
@samthor what would you want to appear in `<title>`? We could look into having Meta.js html escape stuff?

Reply:
ok I tested it and I think it _may_ be ok.

Here's what I got when I did this:

![image](https://user-images.githubusercontent.com/1066253/133709026-93792b37-81b2-462c-8e39-63192f4f97fc.png)

```
<h1 class="course-headline">Learn CSS <code>&lt;script&gt;alert('hello');&lt;/script&gt;</code></h1>
```

The `{{ title | md | safe }}` thing is what we already do for blog posts and (so far at least) it hasn't created any issues.